client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-credentials/hooks/use-credentials-form.ts

```diff
@@ -165,11 +167,15 @@ export const useCredentialsForm = (
 				} );
 				onSubmit( siteInfoResult );
 			} else {
-				const applicationPasswordsInfoResult = await getApplicationPasswordsInfo( siteId, from );
+				const applicationPasswordsInfoResult = await getApplicationPasswordsInfo(
+					siteId,
+					from,
+					locale
+				);
```

Comment:
I think we should check the locale before using it. To ensure it has the appropriate value.

Reply:
I think `locale` should be OK. Looking at the `useLocale` code, it appears to always return a value (even if it defaults to `en`).